src/doc/src/reference/unstable.md

```diff
@@ -1125,32 +1125,6 @@ The `-Z future-incompat-report` flag enables the creation of a future-incompat r
 for all dependencies. This makes users aware if any of their crate's dependencies
 might stop compiling with a future version of Rust.
 
-### configurable-env
```

Comment:
Can you add a redirect at the bottom of the page (in the script tag) for the `#configurable-env` fragment? The target should be "config.html#env"